Lesson_5/index.html

```diff
@@ -2,19 +2,43 @@
 <html>
     <head>
         <link rel="icon" type="image/x-icon" href="https://cdn-icons-png.flaticon.com/512/119/119598.png">
-        <title>Lesson_5 - version 5 main</title>
+        <title>Lesson 5 - version 5.1</title>
     </head>
     <body>
-        <h1>Lesson_5</h1> 
-        <p>Hello World</p> 
-        <br>
-        <br>
-        <strong>Feature A</strong>
+        <h1>Veronica Rubliuk</h1>
+        <img src="img.jpg" width="210" height="270" alt="lesson_5_robot.dreams_rub_ver">
+        <h2>Junior Front-End Developer</h2>
+        <u>
```

Comment:
no block element inside of inline element
so have to be in different way:
`<h3><<u>...</u></h3>`